src/Psl/Type/Internal/PositiveIntType.php

```diff
@@ -47,6 +47,7 @@ public function coerce(mixed $value): int
                 return $int;
             }
 
+            /** @psalm-suppress MissingThrowsDocblock */
```

Comment:
This should result in a `CoerceException` instead of a `InvariantViolationException` I suppose

Reply:
Nice catch, changing it!